src/components/input-section.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export const InputSection: React.FC<{ label: string, children: React.ReactNode[] }> = ({ label, children }) => {
+  return (
+    <div className="bl bw2 br2 b--teal-muted pl2 mb3">
```

Comment:
```suggestion
    <div className="bl bw2 br2 b--teal-muted pl3 mb3">
```
Improves readability 

Before:
![Screenshot 2024-10-24 at 1 51 14 PM](https://github.com/user-attachments/assets/f472aac3-f7e7-4821-a330-29f82ff32368)
After 
![Screenshot 2024-10-24 at 1 51 25 PM](https://github.com/user-attachments/assets/0e384edb-8929-4b38-ba9a-07771de261ca)